python/cugraph/cugraph/tests/test_compat_algo.py

```diff
@@ -0,0 +1,34 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import cugraph.experimental.compat.nx as nx
+
+
+def test_connectivity():
+    expected = list([{1, 2, 3, 4, 5}, {8, 9, 7}])
+    G = nx.Graph()
+    G.add_edges_from([(1, 2), (2, 3), (3, 4), (4, 5)])
+    G.add_edges_from([(7, 8), (8, 9), (7, 9)])
+    assert list(nx.connected_components(G)) == expected
+
+
+def test_pagerank_result_type():
```

Comment:
You have a separate test file `test_compat_pr.py` where you are testing `pagerank`. Why not add `test_pagerank_result_type` there? is there a reason to have the pagerank tests in different files? what is `test_compat_algo.py` meant  for? to hold light tests for all algos?